frontend/src/mocks/appConfigurationHandlers.js

```diff
@@ -30,7 +30,7 @@ export const appConfigurationHandlers = [
 						datetime_format_display: 'August 05 2006, 3:05 PM',
 						logo: '',
 						fav_icon: '',
-						extra_config: null,
+						extra_config: {"git_config":{"branch":{"dev":"dev","prod":"main","staging":"staging"},"repo_url":"https://github.com/Healthlane-Technologies/Zango"},"sync_packages":true},
```

Comment:
@adhiraj23zelthy are these changes intentional?

Reply:
Yes, it will help us to test sync package functionality in frontend using mock API.